isolate.sh

```diff
@@ -801,16 +803,18 @@ function wait_for_certmanager() {
 
 function check_certmanager_count(){
     info "Verifying cert manager is deployed"
-    csv_count=$(${OC} get csv -A | grep "cert-manager"| wc -l | tr -d " " || echo "")
+    csv_count=$(${OC} get csv -A | awk '{print $2}' | grep "cert-manager"| wc -l | tr -d " " || echo "")
     debug1 "cert manager csv count output: $csv_count"
     if [[ "$csv_count" == "0" ]] || [[ "$csv_count" == "" ]]; then
         error "Missing a cert-manager"
     fi
     # if installed in all namespace mode or alongside cp2 cert manager, 
     # csv_count will be >1, need to check for multiple deployments
-    if [[ $csv_count > 1 ]]; then
-        webhook_deployments=$(${OC} get deploy -A --no-headers --ignore-not-found | grep "cert-manager-webhook" -c)
-        if [[ $webhook_deployments != "1" ]]; then
+    if [[ $csv_count > 1 ]]; then 
+        webhook_deployments=$(${OC} get deploy -A --no-headers --ignore-not-found | grep "cert-manager-webhook" -c || echo "")
+        if [[ $webhook_deployments == ""]]; then
+            error "Error happened when trying to get cert-manager csv"
```

Comment:
```suggestion
            error "Failed to cert-manager-webhook deployment"
```

I believe that when command `grep` did not found any matched result, AKA find 0 match line, the command will return non-zero code, causing the error.